experiments/AtmosLES/bomex_model.jl

```diff
@@ -511,6 +511,7 @@ function bomex_model(
         problem = problem,
         turbulence = SmagorinskyLilly{FT}(C_smag),
         moisture = moisture,
+        ref_state = NoReferenceState(),
```

Comment:
This will likely result in CI failing. Probably need to revert this. If you need `NoReferenceState` for a specific experiment, maybe we can pass it in as a kwarg and set the `AtmosModel`'s default (`HydrostaticState(DecayingTemperatureProfile{FT}(param_set),)`).